xbmc/input/ButtonTranslator.cpp

```diff
@@ -1195,6 +1195,38 @@ CAction CButtonTranslator::GetGlobalAction(const CKey &key)
   return GetAction(-1, key, true);
 }
 
+bool CButtonTranslator::HasLonpressMapping(int window, const CKey &key)
+{
+  map<int, buttonMap>::const_iterator it = m_translatorMap.find(window);
+  if (it == m_translatorMap.end())
+  {
+    if (window > -1)
+      return HasLonpressMapping(GetFallbackWindow(window), key);
```

Comment:
@FernetMenta You are right, there is a special handling in `GetFallbackWindow()` for addon windows.
@koying If you had adujsted the method like I said, it would have worked :smile: 

```
if (window > -1)
{
  // first check if we have a fallback for the window
  int fallbackWindow = GetFallbackWindow(window);
  if (fallbackWindow > -1 && IsPossibleLongpress(fallbackWindow, key))
    return true;

  // fallback to default section if there is no key mapping found
  return IsPossibleLongpress(-1, key);
}
```

@FernetMenta could you please test 441582eb2d6e258fcd74c483571abcbe64f02c0b, it should fix the issue. If so I'll create a PR.